UserManagementTeam/src/index.css

```diff
@@ -1,10 +1,11 @@
 body {
   font-family: Arial, Helvetica, sans-serif;
+  background: linear-gradient(135deg, #71b7e6, #9b59b6);
 }
-
-.container {
-  font-size: 3rem;
-  margin: auto;
-  max-width: 800px;
-  margin-top: 20px;
+.custom-card-shadow {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+.custom-label-border {
+  border-image-slice: 1;
+  border-image-source: linear-gradient(135deg, #71b7e6, #9b59b6); 
```

Comment:
there is shadow in tailwind and daisyUI you dont need to write css

Reply:
done